rules/rbe_repo.bzl

```diff
@@ -366,7 +365,10 @@ def _use_standard_config(ctx):
 # Pulls an image using 'docker pull'.
 def _pull_image(ctx, image_name):
     print("Pulling image %s." % image_name)
-    result = ctx.execute(["docker", "pull", image_name])
+
+    # Use l.gcr.io registry to pull marketplace.gcr.io images to avoid auth
+    # issues for users who do not do gcloud login.
+    result = ctx.execute(["docker", "pull", image_name.replace("marketplace.gcr.io", "l.gcr.io")])
```

Comment:
are we sure that all containers in marketplace.gcr.io are mirrored in l.gcr.io (i.e., not just rbe_ubuntu)?

Reply:
Yes.